docs/docs/installation/github.md

```
@@ -50,39 +49,39 @@ When you open your next PR, you should see a comment from `github-actions` bot w
-
-     For example, if your GitHub server is at `https://github.mycompany.com`, add the following to your workflow file:
-     ```yaml
-           env:
-             # ... previous environment values
-             GITHUB.BASE_URL: "https://github.mycompany.com/api/v3"
-     ```
+### Action for GitHub enterprise server 
+!!! tip ""
+    To use the action with a GitHub enterprise server, add an environment variable `GITHUB.BASE_URL` with the API URL of your GitHub server.
+    
+    For example, if your GitHub server is at `https://github.mycompany.com`, add the following to your workflow file:
```

Markdownlint

Expected: fenced; Actual: indented
Code block style

(MD046, code-block-style)